### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sobolevn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,69 +15,90 @@ Could be run on earlier versions of Django, but expect some missing imports warn
...
+
+## FAQ
+
+> Is this an official Django project?
+
+No, it is not. We are indendepent from Django at the moment.
+There's a [proposal](https://github.com/django/deps/pull/65) to merge our project into the Django itself.
+You show your support by linking the PR.
+
+> Is is safe to use this in production?
```
Comment (kalekseev) [inline README.md]: typo: is is

Comment (sobolevn) [reply README.md]: Thanks!

File: README.md [hunk 2]
```diff
@@ -17,69 +15,90 @@ Could be run on earlier versions of Django, but expect some missing imports warn
...
+
+| django-stubs | mypy version | django version | python version
+| ------------ | ---- | ---- | ---- |
+| 1.3.0 | 0.750 | 2.2.x | ^3.6
+| 1.2.0 | 0.730 | 2.2.x | ^3.6
+| 1.1.0 | 0.720 | 2.2.x | ^3.6
+| 0.12.x | old semantic analyzer (<0.711), dmypy support | 2.1.x | ^3.6
+
+
+## FAQ
```
Comment (kalekseev) [inline README.md:56]: Today @mkurnikov provided a good explanation on how to add types to a custom field https://gitter.im/mypy-django/Lobby?at=5e38870140da694c5e06a6ef, it looks like a good candidate for FAQ, another one would be "how to make mypy use custom User model in request.user" the answer is to override original request type with:

```
class MyRequest(Request):
    user: MyUser
```
and use it throughout codebase.

Comment (sobolevn) [reply README.md:56]: Great feedback! Thanks!

Comment (sobolevn) [reply README.md:56]: Done!

**Comment by sobolevn** [comment]
Ok, I am merging this. We can add more `FAQ` entries later.

**Pull request merged**